packages/form-layout/src/vaadin-form-layout-styles.js

```diff
@@ -55,7 +55,8 @@ export const formLayoutStyles = css`
 
   :host([auto-responsive]) #layout {
     display: grid;
-    grid-template-columns: repeat(auto-fit, var(--_column-width));
+    grid-template-columns: repeat(auto-fill, var(--_column-width));
```

Comment:
On my tests, changing from `auto-fit` to `auto-fill` fixes one issue when, eg., the number of columns is defined by an element with a big colspan. For example, a form-layout defined like this:

```html
<vaadin-form-layout auto-responsive>
  <vaadin-form-row>
    <input placeholder="First name" />
    <input placeholder="Last Name" />
  </vaadin-form-row>
  <vaadin-form-row>
		<input placeholder="Address" colspan="4" />
	</vaadin-form-row>
</vaadin-form-layout>
```

Reply:
Could you please check if it would still work with expandColumns in Dialog?

Reply:
I tested it with the changes in `expandColumns` and it seems to work (I made the fields expand to illustrate better):



https://github.com/user-attachments/assets/2c7229bd-c38c-49a5-8f14-50db0b629456